fe/fe-core/src/main/java/com/starrocks/sql/parser/SqlParser.java

```diff
@@ -102,10 +102,18 @@ public static List<StatementBase> parse(String originSql, long sqlMode) {
         return parse(originSql, sessionVariable);
     }
 
+    /**
+     * Please use {@link #parse(String, SessionVariable)}
+     */
```

Comment:
use `parseOneWithStarRocksDialect` ?

Reply:
better to use parse, `parseOneWithStarRocksDialect` only used in special feature